cases/models.py

```diff
@@ -367,6 +370,15 @@ def _timeline(self, actions, action_fn, complaints, history_to_show, user=None):
                 "summary": action_fn(action),
                 "action": action,
             }
+            row["files"] = []
```

Comment:
We can skip the setting of files if history_to_show is not all, just to prevent the data even being made available to a template (and quicker for non-staff then, too).